### ISSUE TITLE
Provide visibility of information in all repository elements

### ISSUE THREAD

**Issue by mss1010** [corona-warn-app/cwa-documentation#402]
## Avoid duplicates
* [x] Question is not already answered in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Question has not already been asked in another issue

## Your Question

In some elements, like PRs, SAP internal jira issues are referenced. But the public does not have read access to this SAP jira workspace. This in itself is fine I guess.
But in this github repository, it should be ensured that everyone has access to the relevant information. So, if an internal issue is mentioned or referenced, additional information about the contents of these issues should be provided, so that the information available on github is transparent and consistent.

To make this abstract description more clear, here is an example: In PR corona-warn-app/cwa-app-ios#883 the description consists purely of references to two internal SAP jira issues. So, for a github user, it´s not clear what this PR does.

So, to summarize, in some way it should be ensured, that all relevant information is visible to any github user.

**Comment by GPclips** [comment]
Hello @mss1010,

thank you for reaching out. I will discuss this topic internally and get back to you with an answer as soon as possible.

Thanks,
LMM

Corona-Warn-App Open Source Team

**Comment by mss1010** [comment]
Hello @GPclips, are there any news on that?

**Comment by svengabr** [comment]
Hello @mss1010, unfortunately, it is not possible to have public access to the SAP Jira. However we, the open source team, are always taking a look at the attached Jira issues to make sure you always have all the information available in the open issues.

Best regards,
SG

Corona-Warn-App Open Source Team

**Comment by daimpi** [comment]
Another more recent example where only a jira issue is mentioned can be found here: https://github.com/corona-warn-app/cwa-app-android/pull/1048

**Comment by mss1010** [comment]
Maybe in such cases, the developers could copy and paste the main text of the jira issue, or write a summary?

**Comment by GPclips** [comment]
As @svengabr mentioned, we always try our very best to add all the required information. I will discuss this topic internally again to make sure all of our members are aware of it. 
I will go on and close the ticket. It can be reopened, if it is needed.

**Comment by Ein-Tim** [comment]
@svengabr 

Hey Sven, could you maybe raise this issue again. It has got a bit worse, I think:
- https://github.com/corona-warn-app/cwa-app-android/pull/5272 only mentions the JIRA ticket
- https://github.com/corona-warn-app/cwa-app-android/pull/5273 only mentions the JIRA ticket
- https://github.com/corona-warn-app/cwa-app-android/pull/5269 only mentions the JIRA ticket 

and so on, I think you see the problem. Just a little description for the community would be great. This is not only a service for the community, but also security relevant, as this brings more trust that PRs don't implement something strange.

Thank you!

**Comment by svengabr** [comment]
I have a call with the developers tomorrow and will ask them to put more information to the pull requests again.

**Comment by Ein-Tim** [comment]
Sadly nothing changed here:

- https://github.com/corona-warn-app/cwa-app-android/pull/5312 only mentions the JIRA ticket
- https://github.com/corona-warn-app/cwa-app-android/pull/5311 only mentions the JIRA ticket and all text in the OP does not make sense without access to the JIRA ticket
- https://github.com/corona-warn-app/cwa-app-android/pull/5236 is better, but could still provide some more details
